telegram/_bot.py

```diff
@@ -4644,32 +4531,22 @@ async def promote_chat_member(
             :class:`telegram.error.TelegramError`
 
         """
-        data: JSONDict = {"chat_id": chat_id, "user_id": user_id}
-
-        if is_anonymous is not None:
-            data["is_anonymous"] = is_anonymous
-        if can_change_info is not None:
-            data["can_change_info"] = can_change_info
-        if can_post_messages is not None:
-            data["can_post_messages"] = can_post_messages
-        if can_edit_messages is not None:
-            data["can_edit_messages"] = can_edit_messages
-        if can_delete_messages is not None:
-            data["can_delete_messages"] = can_delete_messages
-        if can_invite_users is not None:
-            data["can_invite_users"] = can_invite_users
-        if can_restrict_members is not None:
-            data["can_restrict_members"] = can_restrict_members
-        if can_pin_messages is not None:
-            data["can_pin_messages"] = can_pin_messages
-        if can_promote_members is not None:
-            data["can_promote_members"] = can_promote_members
-        if can_manage_chat is not None:
-            data["can_manage_chat"] = can_manage_chat
-        if can_manage_video_chats is not None:
-            data["can_manage_video_chats"] = can_manage_video_chats
-        if can_manage_topics is not None:
-            data["can_manage_topics"] = can_manage_topics
+        data: JSONDict = {
+            "chat_id": chat_id,
+            "user_id": user_id,
+            "is_anonymous": is_anonymous,
+            "can_change_info": can_change_info,
+            "can_post_messages": can_post_messages,
+            "can_edit_messages": can_edit_messages,
+            "can_delete_messages": can_delete_messages,
+            "can_invite_users": can_invite_users,
+            "can_restrict_members": can_restrict_members,
+            "can_pin_messages": can_pin_messages,
+            "can_promote_members": can_promote_members,
+            "can_manage_chat": can_manage_chat,
+            "can_manage_video_chats": can_manage_video_chats,
+            "can_manage_topics": can_manage_topics,
```

Comment:
these and similar lines really create a temptation to use `locals()` :)

Reply:
yep, only have to remove "self" from locals()

at least I had copilot to write these for me, which should reduce typos